tensorflow/python/ops/random_ops.py

```diff
@@ -37,6 +38,16 @@
 from tensorflow.python.util import deprecation
 from tensorflow.python.util.tf_export import tf_export
 
+def _ShapeTensor(shape):
```

Comment:
I'm confused, I don't see where this is called

Reply:
@alextp Looks like the issue was fixed separately by another commit  c4f40ae. Commit  c4f40ae changes the `_ShapeTensor` to `tensor_util.shape_tensor` and that caused this PR to overlap and missed the change after rebase.